lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -61,6 +64,17 @@ def to_html_iruby(placeholder=random_canvas_id)
       high_chart_iruby(extract_chart_class, placeholder, self)
     end
 
+    # @return load the dependent modules of the chart
```

Comment:
At least Input parameter and return object must be documented according to yard.